.github/workflows/build-package.yml

```
@@ -0,0 +1,66 @@
+name: Build Package
+
+env:
+  MAIN_BRANCH: "all-pgautofailover-enterprise"
+  PACKAGE_CLOUD_REPO_NAME: "sample"
+  PACKAGE_ENCRYPTION_KEY: ${{ secrets.PACKAGE_ENCRYPTION_KEY }}
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PACKAGE_CLOUD_API_TOKEN: ${{ secrets.PACKAGE_CLOUD_API_TOKEN }}
+  PACKAGING_SECRET_KEY: ${{ secrets.PACKAGING_SECRET_KEY }}
+  PACKAGING_PASSPHRASE: ${{ secrets.PACKAGING_PASSPHRASE }}
+on:
+  push:
+    branches:
+      - "**"
+
+  workflow_dispatch:
+
+jobs:
+  build_package:
+    name: Build package
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - el/7
+          - el/8
+          - ol/7
+          - debian/stretch
+          - debian/buster
+          - ubuntu/bionic
+          - ubuntu/focal
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Clone tools branch
+        run: git clone -b v0.8.1 --depth=1 https://github.com/citusdata/tools.git tools
+
+      - name: Install package dependencies
+        run: sudo apt-get update && sudo apt install libcurl4-openssl-dev libssl-dev
+
+      - name: Install python requirements
+        run: sudo apt-get update && python -m pip install -r tools/packaging_automation/requirements.txt
```

```suggestion
        run: python -m pip install -r tools/packaging_automation/requirements.txt
```

no need to do this again.

---

Fixed